1_assignment/alphbet_frequency.js

```diff
@@ -0,0 +1,33 @@
+// Sample String .
```

Comment:
Clean up all the mess like Comments and Do Proper indentation.
If Indentation was a person, you simply killed it.